src/App.js

```diff
@@ -28,7 +28,7 @@ import InvoicesList from './components/InvoicesList/InvoicesList';
 import ExclusiveForm from './components/DopplerPlus/ExclusiveForm';
 import { ContactPolicy } from './components/ContactPolicy/ContactPolicy';
 import { Promotional } from './components/Integrations/BigQuery/Promotional/Promotional';
-import { AuthorizationPage } from './components/Integrations/BigQuery/ControlPanel/AuthorizationPage';
+import AuthorizationPage from './components/Integrations/BigQuery/ControlPanel/AuthorizationPage';
```

Comment:
I think that we are trying to avoid usage of _default exports_, see the rationale here: https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/

Reply:
when using the dependency injector is the way to export the class in the solution, i can make this change, but it would not be consistent with the rest of the components of the solution, do you agree with this change

Posible change 
App.js
![imagen](https://user-images.githubusercontent.com/11965643/125796069-8db6d9fb-f691-431c-9b9b-64eca4d7a134.png)

new way to export 

![imagen](https://user-images.githubusercontent.com/11965643/125796177-1d07dc7a-283c-43a6-b623-0cef5a71610c.png)


do you agree with this change ?

Reply:
Oh, I understand. @cbernat @jcamposmk what is your opinion?

Reply:
Why not do this?
```
export const AuthorizationPage = InjectAppServices ( ( {dependencies: bigQueryClient .... 
```